docs/notebooks/proton_radiography_particle_tracing.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
<p>Eventually I think it'd make a ton of sense to have <code>prad.test_fields</code>  return some sort of <code>Plasma</code>  object - if it's just a grid with an electric and magnetic fields... I think this might finally push me to reimplement  our gridded<code>Plasma</code> class through xarray. It'd simplify this nicely. Nothing to worry about; just pointing this out for future reference.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Sure, that sounds great! If there is a gridded plasma object that includes a spatial grid as well as arrays of E and B, that type could also be used as input for the SimPrad class.

Reply:
I discussed this with Erik yesterday at the telecon; it's a decent enough idea but let's do it this way for now and adjust it later. :)